cmd/cmount/mountpoint_windows.go

```diff
@@ -132,30 +137,47 @@ func handleLocalMountpath(f fs.Fs, mountpath string, opt *mountlib.Options) (str
 	return mountpath, nil
 }
 
+// networkSharePathEncoder is an encoder used to make strings valid as (part of) Windows network share UNC paths
+const networkSharePathEncoder = (encoder.EncodeZero | // NUL(0x00)
```

Comment:
I would have thought you could put the encoder and `encodeNetworkSharePath` into mountlib and make this patch a lot neater. I think then it would only be in mountlib.